19/q10-19-yona.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+int main()
+{
+    int n,i,sum;
+    sum=0;
+    printf("please enter no to check for prime no");
+    scanf("%d",&n);
+    for(i=2;i<=n/2;i++)

```

Comment:
Good, using for loop to run upto n/2.
And yes you can improve the efficiency of the program by changing n/2 by the sqrt(n) so the loop will run upto the square root of a number, so it will run faster when checking very large numbers